kloppy/infra/serializers/event/statsbomb/deserializer.py

```diff
@@ -175,14 +178,27 @@ def create_teams_and_players(self, raw_events, lineups):
         )
 
         # Create players and teams
-        player_positions = {
-            str(player["player"]["id"]): Position(
-                position_id=str(player["position"]["id"]),
-                name=player["position"]["name"],
-            )
-            for raw_event in starting_xi_events
-            for player in raw_event["tactics"]["lineup"]
-        }
+        team_position_changes = {}
+        for lineup in lineups:
+            for player in lineup["lineup"]:
+                position_changes = []
+                for position in player["positions"]:
+                    start_timestamp = parse_football_clock_str_ts(
+                        position["from"]
+                    )
+                    position_changes.append(
+                        {
+                            "timestamp": start_timestamp,
+                            "period_id": position["from_period"],
+                            "position": Position(
+                                position_id=str(position["position_id"]),
+                                name=position["position"],
+                            ),
+                        }
```

Comment:
I think it is a bit against the "philosophy" of kloppy to use a dict here. Instead, you could create a `PositionChange` object. I also wonder whether it makes sense to use a `PositionChangeEvent`  for this and also add these as events to the event stream. What do you think @janvanhaaren?